doc/fluxing.rst

```diff
@@ -12,6 +12,105 @@ It is a two step process of generating a `Sensitivity Function`_ and then
 `Applying the Sensitivity Function`_.  We describe each in turn.  However, we
 first discuss the basic fluxing concepts/approach.
 
+
+Sensitivity Function
+====================
+
+The sensitivity function is generated from the :doc:`out_spec1D` file of a
+processed standard star.
+
+``PypeIt`` uses an archived fluxed spectrum from either the `CALSPEC calibration
+database <http://stsci.edu/hst/observatory/crds/calspec.html>`_ or one of the
+files we have grabbed from `ESO
+<https://www.eso.org/sci/observing/tools/standards/spectra/stanlis.html>`_.  If
+you observed something else, see `Adding a Standard Star`_.
+
+The sensitivity function is generated by dividing the standard star's flux in
+units of 1e-17 erg/s/cm^2/Ang by the standard star's counts per second per
+Angstrom [photons/s/Ang]. The sensitivity function is written to disk as a FITS
```

Comment:
should this be "photons" or "electrons"?  I.e., have we removed QE?

Reply:
I think electrons.  Have modified